src/OddJobs/ConfigBuilder.hs

```diff
@@ -63,14 +63,15 @@ mkConfig logger tname dbpool ccControl jrunner configOverridesFn =
             , cfgDbPool = dbpool
             , cfgOnJobStart = (const $ pure ())
             , cfgDefaultMaxAttempts = 10
-            , cfgTableName = tname
+            , cfgTableNames = simpleTableNames tname
             , cfgOnJobTimeout = (const $ pure ())
             , cfgConcurrencyControl = ccControl
+            , cfgDefaultResourceLimit = 1
```

Comment:
My thought is that `1` (meaning jobs with a resource with no limit would "allow execution with minimal impact") is a reasonable default. The other more or less reasonable defaults are `0` (meaning these jobs would never run) and `maxBound` (meaning there would effectively be no limit to the concurrency for these jobs).   Thoughts?